_automating-configurations/api/delete-workflow.md

```diff
@@ -50,4 +65,6 @@ If the workflow exists, a delete response contains the status of the deletion, w
   "_seq_no": 2,
   "_primary_term": 1
 }
-```
\ No newline at end of file
+```
+
+Note: If the `clear_status` parameter was `true`, also deletes the Workflow State if the provisioning status is any state other than `IN_PROGRESS`.
```

Comment:
I don't think this is needed because it's explained in the parameter table. We're just providing an example response here.